src/Makefile.am

```diff
@@ -0,0 +1,4 @@
+lib_LTLIBRARIES=libmo_unpack.la
+libmo_unpack_la_SOURCES=convert_float_ibm_to_ieee32.c convert_float_ieee32_to_ibm.c extract_bitmaps.c extract_nbit_words.c extract_wgdos_row.c logerrors.c logerrors.h pack_ppfield.c read_wgdos_bitmaps.ibm.c rlencode.c rlencode.h uascii.c unpack_ppfield.c wgdos_decode_field_parameters.c wgdos_decode_row_parameters.c wgdos_expand_row_to_data.c wgdos_pack.c wgdosstuff.h wgdos_unpack.c
+libmo_unpack_la_LDFLAGS=-version-info 2:5:0
```

Comment:
It may be useful to add some variables defined in `configure.ac` and used here. That way, the release manager only has one file to change (for version + .so version) instead of two.